algorithm_toolkit/views/home.py

```diff
@@ -131,106 +132,123 @@ def show_docs(filename):
     return send_from_directory(docs_path, filename)
 
 
-@home.route('/main/', methods=['POST', 'GET'])
+# Chain name not in the request form, but in the endpoint!
+# New namespace ( /api/ ) added to prevent user from envoking other static routes
+@home.route('/api/<chain_name>/', methods=['POST', 'GET'])
```

Comment:
Maybe we can use /chains/<chain_name> here? That could allow us to (in the future) expand the API to provide simply a "/chains" endpoint which would return a collection (list) of chains (standard REST api organization)? @chriswilley, thoughts?

Reply:
Change made